docker/mountFolder/main.dml

```diff
@@ -19,4 +19,15 @@
 #
 #-------------------------------------------------------------
 
-print("Hello, World!");
\ No newline at end of file
+# kmeans_test.dml
+# Simple KMeans clustering test
+
+# Arguments
+X = read("test.bin")
```

Comment:
This will only work if there is a test.bin file. Please change back to the in place construction of the matrix.